### PR TITLE
Create artifacts.md

### PR DESCRIPTION
**Pull request by KrauseFx**
Fixes https://github.com/fastlane/ci/issues/169

### CODE REVIEW

**Comment by KrauseFx** [comment]
Let me know if y'all prefer a different file name

File: docs/use_cases/artifacts.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    1. the used storage backend (e.g. Google Cloud Storage, SFTP, etc.)
+    1. the name of the artifact
+1. By clicking on the specific build artifact, a new browser tab is opened with the URL provided by the storage used for this particular build
+
+### Error states ###
+[WIP]
+
+### Notes ###
+
+The prettified output is generated by _fastlane_ using [xcpretty](https://github.com/supermarin/xcpretty). Both _gym_ and _scan_ also generate a raw `.log` file that doesn't use `xcpretty`. This is :key:, as sometimes `xcpretty` would swallow relevent error information.
```
Comment (taquitos) [inline docs/use_cases/artifacts.md]: Ok, so both will be included (prettified and original)? If so 💯

Comment (KrauseFx) [reply docs/use_cases/artifacts.md]: Yes, that's the idea, should I add an extra sentence? I appended an extra commit also that mentions it kind of https://github.com/fastlane/ci/pull/178/files#diff-12fdd14c1e682d3cd25c7f56161c8eebR8

**Comment by taquitos** [comment]
How do you feel about `download_build_artifacts.md`
Also, would it make sense to call `builds` `jobs` instead? `build` implies packaging something, while `job` implies generic action taken.

File: docs/use_cases/artifacts.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Artifacts #
+
+Accessing build artifacts is an essential piece of CI systems. 
+`fastlane.ci` supports a flexible system to allow any kind of store options, so that developers can own their data.
+For more information on how it works, check out [docs/ArtifactsAndBuilds.md](/docs/ArtifactsAndBuilds.md)
+
+Artifacts include:
+- the complete build output (both prettified, and the raw output)
```
Comment (taquitos) [inline docs/use_cases/artifacts.md]: 🚀

**Pull request merged**